controller/src/test/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessorsTest.java

```diff
@@ -90,13 +91,57 @@ public void testEventKey() {
         assertEquals(commitEvent.getKey(), "test/test");
     }
 
+    @Test(timeout = 10000)
+    public void testIsReady() {
+        Controller localController = mock(Controller.class);
+        CheckpointStore checkpointStore = mock(CheckpointStore.class);
+        StreamMetadataStore streamStore = mock(StreamMetadataStore.class);
+        BucketStore bucketStore = mock(BucketStore.class);
+        ConnectionPool connectionPool = mock(ConnectionPool.class);
+        StreamMetadataTasks streamMetadataTasks = mock(StreamMetadataTasks.class);
+        StreamTransactionMetadataTasks streamTransactionMetadataTasks = mock(StreamTransactionMetadataTasks.class);
+        KVTableMetadataStore kvtStore = mock(KVTableMetadataStore.class);
+        TableMetadataTasks kvtTasks = mock(TableMetadataTasks.class);
+        ControllerEventProcessorConfig config = ControllerEventProcessorConfigImpl.withDefault();
+        EventProcessorSystem system = mock(EventProcessorSystem.class);
+        CuratorZookeeperClient curatorZKClientMock = mock(CuratorZookeeperClient.class);
+        CuratorFramework client  = mock(CuratorFramework.class);
+        when(checkpointStore.isHealthy()).thenReturn(true);
+        @Cleanup
+        ControllerEventProcessors processors = spy(new ControllerEventProcessors("host1",
+                config, localController, checkpointStore, streamStore, bucketStore,
+                connectionPool, streamMetadataTasks, streamTransactionMetadataTasks,
+                kvtStore, kvtTasks, system, executorService()));
+        Assert.assertFalse(processors.isReady());
+        doReturn(true).when(processors).isBootstrapCompleted();
```

Comment:
This should be unnecessary after we invoke `processors.bootstrap()`

Reply:
Thanks for pointing out. This was unnecessary. Removed now